docs/dokka/-kover -gradle -plugin/kotlinx.kover.gradle.plugin.dsl/-kover-html-report-config/index.html

```diff
@@ -58,24 +58,24 @@
   <div class="breadcrumbs"><a href="../../../index.html">Kover Gradle Plugin</a><span class="delimiter">/</span><a href="../index.html">kotlinx.kover.gradle.plugin.dsl</a><span class="delimiter">/</span><span class="current">KoverHtmlReportConfig</span></div>
   <div class="cover ">
     <h1 class="cover"><span>Kover</span><wbr></wbr><span>Html</span><wbr></wbr><span>Report</span><wbr></wbr><span><span>Config</span></span></h1>
-    <div class="platform-hinted " data-platform-hinted="data-platform-hinted"><div class="content sourceset-dependent-content" data-active="" data-togglable=":dokkaHtml/main"><div class="symbol monospace"><span class="token keyword">interface </span><a href="index.html">KoverHtmlReportConfig</a> : <a href="../-kover-general-html-report-config/index.html">KoverGeneralHtmlReportConfig</a></div></div></div>
+    <div class="platform-hinted " data-platform-hinted="data-platform-hinted"><div class="content sourceset-dependent-content" data-active="" data-togglable=":dokkaHtml/main"><div class="symbol monospace"><span class="token keyword">interface </span><a href="index.html">KoverHtmlReportConfig</a></div></div></div>
```

Comment:
Hmm why we have html files in `docs` folder instead of separate gh-pages branch?

Reply:
Automatic actions for the `gh-pages` branch are not configured right now.

It is supposed to publish pages from the `release` branch.
Perhaps in the future it is worth setting up the automatic launch of the `dokkaHtml` task.